reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Skip.kt

```diff
@@ -0,0 +1,33 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.completable.CompletableCallbacks
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+
+fun <T> Observable<T>.skip(count: Long): Observable<T> =
+    observable { emitter ->
```

Comment:
You can use `observableUnsafe { observer`, it's safe for such a simple operator. We use `observable { emitter` when there is a chance you will call callbacks in inappropriate order.

Reply:
Fixed it 👍